### PR TITLE
Course Enrollment Setting Page

### PR DESCRIPTION
**Pull request by xinnanlisf**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: force-app/main/default/lwc/programSettings/programSettings.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+            <div class="slds-col slds-size_3-of-3 slds-col_bump-right slds-p-right_x-large">
+                <lightning-datatable
+                    key-field="mappingName"
+                    data={primaryAffiliationsSettingsVModel.primaryAffiliationMappings}
+                    columns={primaryAffiliationMappingsTableColumns}
+                    onrowaction={handlePrimaryAffiliationsRowAction}
+                ></lightning-datatable>
+            </div>
+        </template>-->
```
Comment (MatthewBlanski) [inline force-app/main/default/lwc/programSettings/programSettings.html:43]: Let's remove this comment in this PR.

File: force-app/main/default/lwc/programSettings/programSettings.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import { LightningElement } from "lwc";
+import { refreshApex } from "@salesforce/apex";
+
+import stgProgramsSettingsTitle from "@salesforce/label/c.stgProgramsSettingsTitle";
+
+export default class programSettings extends LightningElement {
```
Comment (MatthewBlanski) [inline force-app/main/default/lwc/programSettings/programSettings.js:6]: Please incorporate the new plumbing for the focus handling bug.

**Review by MatthewBlanski — CHANGES_REQUESTED** [review]
Just one change in incorporating a branch that is currently in progress so we don't further propagate a bug.

File: force-app/main/default/lwc/programSettings/programSettings.html (hunk 1 above)
Comment (xinnanlisf) [reply force-app/main/default/lwc/programSettings/programSettings.html:43]: This works as a placeholder for the setting page but sure i can get rid of it.

**Comment by xinnanlisf** [comment]
@MatthewBlanski should be good now!

**Review by MatthewBlanski — CHANGES_REQUESTED** [review]
The incorporation of the new methods for settings pages remains to be done.

**Comment by xinnanlisf** [comment]
@MatthewBlanski, I have put the auto enrollment mapping navigation custom label in this PR. Also, I have double checked the focus bug fix. It looks like it is working in this PR. Let me know what I miss.

**Review by MatthewBlanski — APPROVED** [review]

**Comment by spinkelman** [comment]
QA Passes - Merging

**Pull request merged**